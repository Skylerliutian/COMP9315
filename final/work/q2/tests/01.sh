./bnl 3 4 5
