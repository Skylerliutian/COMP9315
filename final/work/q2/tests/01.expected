Running: ./bnl 3 4 5

Frames:     [00] [01] [02] [03] [04]
Contents:    S03  R02  S00  S01  S02
PinCount:      0    0    0    0    0
Popularity:    2    2    2    2    2
Clock: 1

#requests: 15
#releases: 15
#hits    : 4
#reads   : 11
